features/mesh/services/mesh-services/Index.feature

```diff
@@ -29,8 +29,8 @@ Feature: mesh / services / mesh-services / index
         When I visit the "<URL>" URL
         Then the "$button-group" element exists
         And I click the "$item a" element
-        Then the URL contains "monitor-proxy-0.kuma-demo"
-        And the URL doesn't contain "monitor-proxy-0.kuma-demo/overview"
+        Then the URL contains "monitor-proxy-0"
+        And the URL doesn't contain "monitor-proxy-0/overview"
```

Comment:
We still need to use the full `name.namespace` in the URL to keep it unique.

Elsewhere we add an `id` property for the resource/object containing the `name.namespace` and we use that as the unique thing for links/URLs etc, and then use the `name` for the thing we render to the page.

https://github.com/kumahq/kuma-gui/blob/9dfdcfac45466501205dd8b5bb1bbc8d60ebe9b7/src/app/zone-ingresses/data/index.ts#L75

https://github.com/kumahq/kuma-gui/blob/9dfdcfac45466501205dd8b5bb1bbc8d60ebe9b7/src/app/zone-ingresses/views/ZoneIngressListView.vue#L67

Reply:
Ah thank you, that makes sense 🙂